http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -298,7 +301,7 @@ class RuntimeServiceInterp[F[_]: Parallel](
         )
         .transaction
 
-    } yield runtimes.toVector
```

Comment:
why remove this toVector call?

Reply:
becuz runtimes is already `Vector`..there's no point calling `toVector` again